tools/release.js

```diff
@@ -23,11 +24,15 @@ startRelease().then(() => {
 
 async function startRelease() {
   // First check we are on master
-  if (branchName() !== 'master') {
-    logError("you shall not pass! You are not on the master branch!")
+  if (branchName() !== branch) {
+    logError("you shall not pass! You are not on the master " + branch + "!")
     return;
   }
 
+  // Since we need to jump between branches, ask the use to make sure branches are clean. 
+  // Maybe we could work with the stash, but we keep it simple for now. It seems logic to have a clean branch before a release.
+  await ask(`Make sure all your changes are commited or the script will fail to update the playground. (type anything to continue)`);
```

Comment:
committed

Reply:
I removed this. Because it should always work as we commit -am later. After the commit everything should be cleaned before the checkout.